### ISSUE TITLE
websit official docs don't access

### ISSUE THREAD

**Issue by shadow1163** [Gagnavarslan/sst#18]
http://testutils.org/sst
websit official docs don't access

**Comment by arnlaugsson** [comment]
@shadow1163, yes it seems as if SST isn't being actively maintained. But the README are in this repo, it's just not being exported to anywhere.
